rocketpy/Function.py

```diff
@@ -2030,3 +2118,101 @@ def differentiate(self, x, dx=1e-6):
         # h = (10)**-300
         # z = x + h*1j
         # return self(z).imag/h
+
+
+def funcify_method(*args, **kwargs):
+    """Decorator factory to wrap methods as Function objects and save them as cached
+    properties.
+
+    Parameters
+    ----------
+    *args : list
+        Positional arguments to be passed to rocketpy.Function.
+    **kwargs : dict
+        Keyword arguments to be passed to rocketpy.Function.
+
+    Returns
+    -------
+    decorator : function
+        Decorator function to wrap callables as Function objects.
+
+    Examples
+    --------
+    >>> class Test():
+    ...     @Funcify(inputs=['x'], outputs=['y'])
```

Comment:
Docstring still references old naming `Funcify` instead of `funcify_method`.

Reply:
Thanks for noticing! I'll fix this right away.